.github/workflows/release.yml

```
@@ -36,3 +48,23 @@ jobs:
       - run: npm publish --access=public --workspace=packages/core
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+  npm-cli:
+    name: npm publish @web3-storage/content-claims-cli
+    needs: release
+    if: force_release_cli || contains(fromJson(needs.release.outputs.paths_released), 'packages/cli')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.3
+        with:
+          version: 8
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build --workspace=packages/cli
+      - run: npm publish --access=public --workspace=packages/cli
```

Let's add build provenance here and to the core lib too! see: https://github.com/web3-storage/w3cli/pull/135
```suggestion
      - run: npm publish --access=public --workspace=packages/cli --provenance
```

note that it will also need permissions defined as per https://github.com/web3-storage/w3cli/pull/135#pullrequestreview-1744911347

---

added with permissions in https://github.com/web3-storage/content-claims/pull/46/commits/63f63963c7bebaf9123a59a011811567e631b361

---

this other PR adds to core lib https://github.com/web3-storage/content-claims/pull/47